src/Orchard.Web/Modules/Orchard.OpenId/Constants.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Orchard.OpenId {
+    public class Constants {
```

Comment:
I think the structure would be nicer if these constants were moved into feature-specific constant files. Since it's a possible extensibility point to write more providers.